### ISSUE TITLE
[1.12.2] Broken copper and tin smelting recipe when using other mods

### ISSUE THREAD

**Issue by theo546** [coolsimulations/SurvivalPlus#8]
Hello,

When using this mod with Mekanism, Industrial Craft 2 and Thermal Foundation, the copper and tin ore are not smelteable in a regular furnace anymore.

This is an unwanted behavior and is making this impossible to acquire copper ingot in a regular manner.

Here is a complete list of the mods I've used to reproduce this bug:
```SurvivalPlus-1.12.2-0.1.5
CodeChickenLib-1.12.2-3.2.3.358-universal
RedstoneFlux-1.12-2.1.1.1-universal
ThermalCultivation-1.12.2-0.3.6.1-universal
ThermalInnovation-1.12.2-0.3.6.1-universal
CoFHCore-1.12.2-4.6.6.1-universal
CoFHWorld-1.12.2-1.4.0.1-universal
ThermalDynamics-1.12.2-2.5.6.1-universal
ThermalFoundation-1.12.2-2.6.7.1-universal
ThermalExpansion-1.12.2-5.5.7.1-universal
Mekanism-1.12.2-9.8.3.390
WanionLib-1.12.2-2.5
UniDict-1.12.2-3.0.8
industrialcraft-2-2.8.221-ex112
```

I am able to reproduce this bug with and without UniDict installed.

I also tried to tweak this behavior using the `Enable Copper and Tin Smelting Recipe Replacement` option without any positive result, the problem is still the same, SurvivalPlus is messing with the copper and tin smelting recipe without any way to get it fixed.

Forge version 14.23.5.2855.

**Comment by coolsimulations** [comment]
Greetings! I am aware of this issue and it will get fixed within the next month. That's strange that the ```Enable Copper and Tin Smelting Recipe Replacement``` option isn't working, it worked as a temporary fix for me... Other than that, I can't think of any workarounds aside from removing IC2 until I release the next update.

**Comment by theo546** [comment]
Alright, thank you very much!

**Comment by coolsimulations** [comment]
Should be fixed with https://github.com/coolsimulations/SurvivalPlus/commit/b157c1e74e95c03909bf90ebd1b4773050fa91d3. Version 0.1.6 will be released when I finish updating to 1.18.
